### DISCUSSION TITLE
Why can't I delete a (cname).github.io repository

### DISCUSSION THREAD

**Discussion by gongzhaoxu** [community/community#29746] [Other Feature Feedback, Questions, & Ideas] [answered]
I create a github.io repository:
![10-56-45-2022-08-17](https://user-images.githubusercontent.com/65840992/185024924-62f766e6-5fb5-4ad7-a8e7-747a663397f3.png)
It's a newly created empty repo, but when I click it, I get a 404 error. Not entering the repo as usual.
![404](https://user-images.githubusercontent.com/65840992/185025210-9eb7feca-416e-4572-a1d3-20f7fcd6229d.png)
So how can i delete it, the repo now is completely unmanageable.

**Comment by radar-blip** [comment, marked as answer]
Open repo, click on settings, and scroll to the bottom. >> Delete

**Reply by radar-blip** [reply]
For future reference, you must include an index.html or readme.md in your repo in order to use github pages (index.html is better choice).
Good luck!

**Reply by gongzhaoxu** [reply]
Problem solved, thanks for your suggestion

**Reply by radar-blip** [reply]
You're welcome! 👍🏼
